src/app/shared/navbar/navbar.scss

```diff
@@ -15,14 +15,42 @@
   flex-grow: 1;
 }
 
-.docs-angular-logo {
-  height: 26px;
-  margin: 0 4px 3px 0;
-  vertical-align: middle;
+.angular-logo {
+  height: 30px;
+  width: 30px;
+}
+
+.docs-navbar {
+  display: none;
+
+  .docs-nav-link {
+    text-decoration: inherit;
+    flex: 1;
+  }
+}
+
+.show-small {
```

Comment:
Missing prefix (here and elsewhere). The prefix is typically something like `docs-navbar-`